ocis/docker/manifest.tmpl

```diff
@@ -1,16 +1,16 @@
-image: owncloud/ocis:{{#if build.tag}}{{trimPrefix "v" build.tag}}{{else}}latest{{/if}}
+image: owncloud/ocis-rolling:{{#if build.tag}}{{trimPrefix "v" build.tag}}{{else}}latest{{/if}}
```

Comment:
I have no idea how docker manifest works so I wonder if this is correct

Reply:
this is fine